cmd/timestamp-server/app/serve.go

```diff
@@ -38,7 +38,9 @@ var serveCmd = &cobra.Command{
 	Short: "start http server with configured api",
 	Long:  `Starts a http server and serves the configured api`,
 	Run: func(cmd *cobra.Command, args []string) {
-
+		if err := viper.BindPFlags(cmd.Flags()); err != nil {
```

Comment:
Flags are bound here - https://github.com/sigstore/timestamp-authority/blob/main/cmd/timestamp-server/app/root.go#L71

Reply:
Yes, for these flags, if you try using any other available flags those are not bound, e.g. --scheme

Reply:
Ah thanks, thought that was for all flags. I’ll take another look Monday!

Reply:
So we should either remove these flags or allow users to set them when desired.